Sources/TuistGenerator/GraphViz/GraphToGraphVizMapper.swift

```diff
@@ -97,7 +97,7 @@ extension GraphDependency {
             return path.basenameWithoutExt
         case let .bundle(path):
             return path.basenameWithoutExt
-        case let .packageProduct(path: _, product: product):
+        case let .packageProduct(path: _, product: product, _):
```

Comment:
Nitpick: I'd keep the name of the attributes even if we are not using them for readability reasons. Otherwise one has to go to the definition of the case to know what `_` means. What do you think?

Reply:
This comment applies to all the changes in this PR :)

Reply:
ah, that makes sense. but since this style is not applied consistently throughout the code base, I only adapted it where it improves consistency (i.e.: here 😄)